relay-general/src/store/normalize.rs

```diff
@@ -293,15 +321,17 @@ fn remove_invalid_measurements(
         }
 
         // Retain payloads in _meta just for excessive custom measurements.
-        if let Some(measurement) = value.value_mut().take() {
-            removed_measurements.insert(name.clone(), Annotated::new(measurement));
-        }
+        excessive_measurements = true;
```

Comment:
since we can no longer rely on `removed_metrics` representing only excessive measurements, i made this flag instead. I don't add the error directly as then we won't get multiple of the same type.